geopyspark/geotrellis/render.py

```diff
@@ -0,0 +1,56 @@
+from geopyspark.geotrellis.constants import NEARESTNEIGHBOR, ZOOM
+
+class PngRDD(object):
```

Comment:
Could you please add docstrings to this file? https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/rdd.py Has good examples of the docstring style.